src/templates/rule.js

```diff
@@ -19,6 +19,8 @@ import Breadcrumb from '../components/breadcrumb/breadcrumb';
 import Acknowledgements from '../components/acknowledgements/acknowledgements';
 import Comments from '../components/comments/comments';
 import Reaction from '../components/reaction/reaction';
+import { faPencilAlt } from '@fortawesome/free-solid-svg-icons';
+import { faGithub } from '@fortawesome/free-brands-svg-icons';
```

Comment:
You can do this in one line
```suggestion
import { faPencilAlt, faGithub } from '@fortawesome/free-solid-svg-icons';
```

Reply:
As per our conversation, you overlooked that they come from two different folders.